### PR TITLE
feat: Move actions call to platform [8]

### PR DESCRIPTION
**Pull request by bistaastha**

### CODE REVIEW

File: src/app/core/services/platform/v1/approver/reports.service.spec.ts [hunk 1]
```diff
@@ -117,6 +118,18 @@ describe('ApproverReportsService', () => {
     });
   });
 
+  it('permissions(): should get report permissions', (done) => {
+    approverPlatformApiService.post.and.returnValue(of({ data: apiReportPermissions }));
+
+    const id = 'rpxtbiLXQZUm';
```
Comment (Chethan-Fyle) [inline src/app/core/services/platform/v1/approver/reports.service.spec.ts:124]: `const id = apiReportPermissions.id;`

File: src/app/core/services/platform/v1/spender/reports.service.spec.ts [hunk 2]
```diff
@@ -67,6 +68,18 @@ describe('SpenderReportsService', () => {
     });
   });
 
+  it('permissions(): should get report permissions', (done) => {
+    spenderPlatformV1ApiService.post.and.returnValue(of({ data: apiReportPermissions }));
+
+    const id = 'rpxtbiLXQZUm';
```
Comment (Chethan-Fyle) [inline src/app/core/services/platform/v1/spender/reports.service.spec.ts:74]: Use variable instead of value --> `const id = apiReportPermissions.id;`

**Review by Chethan-Fyle — APPROVED** [review]

File: src/app/fyle/view-team-report/view-team-report.page.html [hunk 3]
```diff
@@ -292,8 +292,8 @@
   </div>
 </ion-content>
 
-<ng-container *ngIf="actions$|async as actions">
-  <ion-footer *ngIf="!isCommentsView && isReportReported && actions.can_approve" class="cta-footer">
+<ng-container *ngIf="permissions$|async as permissions">
+  <ion-footer *ngIf="!isCommentsView && isReportReported && permissions.can_approve" class="cta-footer">
```
Comment (mvaishnavi) [inline src/app/fyle/view-team-report/view-team-report.page.html:296]: what does `isReportReported` do? 
can we get rid of this?

**Review by mvaishnavi — APPROVED** [review]

File: src/app/fyle/view-team-report/view-team-report.page.html (hunk 3 above)
Comment (bistaastha) [reply src/app/fyle/view-team-report/view-team-report.page.html:296]: Will handle with post api changes

**Pull request closed without merging**